modules/zcbor/Kconfig

```diff
@@ -25,6 +25,7 @@ config ZCBOR_STOP_ON_ERROR
 
 config ZCBOR_VERBOSE
 	bool "Make zcbor code print messages"
+	depends on !MINIMAL_LIBC # Because Minimal LIBC doesn't have PRIuFAST32 and PRIxFAST32.
```

Comment:
If we are going to merge this without fixing the minimal libc, please add `FIXME:` in the comment so it can be easily grepped later.

Reply:
Done